### PR TITLE
Fix Broken Link (doc)

### LINKED ISSUE #705: Broken Link in Documentation > Guides
### Describe the bug

On the following pages, the link to nsxt_policy_group (https://www.terraform.io/docs/providers/nsxt/r/policy_group.html?_ga=2.121573453.435863937.1641316818-1469427522.1597791478) is broken.

* https://registry.terraform.io/providers/vmware/nsxt/latest/docs/guides/vmc
* https://registry.terraform.io/providers/vmware/nsxt/latest/docs/guides/federation

### Reproduction steps

```bash
See bug description, links are broken.
```

### Expected behavior

Guides link to the correct resource.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by alagoutte**
Fix #705

### CODE REVIEW

File: website/docs/guides/federation.html.markdown [hunk 1]
```diff
@@ -40,33 +40,33 @@ Remember to check out the documentation for the resource you are interested in f
...
- * Tier-1 Gateway [nsxt_policy_tier1_gateway](https://www.terraform.io/docs/providers/nsxt/r/policy_tier1_gateway.html)
- * Tier-1 Interface [nsxt_policy_tier1_interface](https://www.terraform.io/docs/providers/nsxt/r/policy_tier1_interface.html)
- * Segment [nsxt_policy_segment](https://www.terraform.io/docs/providers/nsxt/r/policy_segment.html)
- * VLAN Segment [nsxt_policy_vlan_segment](https://www.terraform.io/docs/providers/nsxt/r/policy_vlan_segment.html)
- * Group [nsxt_policy_group](https://www.terraform.io/docs/providers/nsxt/r/policy_group.html)
- * Service [nsxt_policy_service](https://www.terraform.io/docs/providers/nsxt/r/policy_service.html)
- * DFW Security Policy [nsxt_policy_security_policy](https://www.terraform.io/docs/providers/nsxt/r/policy_security_policy.html)
- * Gateway Policy [nsxt_policy_gateway_policy](https://www.terraform.io/docs/providers/nsxt/r/policy_gateway_policy.html)
- * NAT Rule [nsxt_policy_nat_rule](https://www.terraform.io/docs/providers/nsxt/r/policy_nat_rule.html)
+ * Tier-0 Gateway [nsxt_policy_tier0_gateway](https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_gateway)
```
[the remark this answers is not in the document]
Comment (alagoutte) [reply website/docs/guides/federation.html.markdown:43]: th redirect work for me...

[the remark this answers is not in the document]
Comment (annakhm) [reply website/docs/guides/federation.html.markdown:43]: Yes, my bad, most of them work, but I probably landed on bad one - mentioned below

File: website/docs/guides/federation.html.markdown [hunk 2]
```diff
@@ -40,33 +40,33 @@ Remember to check out the documentation for the resource you are interested in f
...
+ * Tier-0 Interface [nsxt_policy_tier0_interface](https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_interface)
```
Comment (annakhm) [inline website/docs/guides/federation.html.markdown:44]: should be https://www.terraform.io/docs/providers/nsxt/r/policy_tier0_gateway_interface, same below for tier1

**Comment by annakhm** [comment]
Thank you! Merging this.

**Pull request merged**